src/libs/reportUtils.js

```diff
@@ -35,6 +45,20 @@ function sortReportsByLastVisited(reports) {
         .value();
 }
 
+/**
+ * Check whether user can edit report action
+ *
+ * @param {Object} reportAction
+ * @param {String} sessionEmail
+ * @returns {Boolean}
+ */
+function canEditReportAction(reportAction) {
+    return reportAction.actorEmail === sessionEmail
```

Comment:
Let's add a comment in here:

```
// Can only edit if it's a ADDCOMMENT, the author is this user and it's not a optimistic response.
        // If it's an optimistic response comment it will not have a reportActionID,
        // and we should wait until it does before we show the actions
```